### ISSUE TITLE
How to open?

### ISSUE THREAD

**Issue by johannes510** [Robert01337/Vanta-External-Remake#1]
Hi, how can i start the project? Shouldn't be there a .sln?

**Comment by Robert01337** [comment]
> Hi, how can i start the project? Shouldn't be there a .sln?

Yes, you have to join our discord --> https://discord.gg/blueberryfn
